devex/vscode-extension/src/common/constants.ts

```diff
@@ -10,6 +10,11 @@ export class Constants {
     //
     public static ExtensionId = "ms-iot.msiot-vscode-openenclave";
 
+    //
+    // SDK version support
+    //
+    public static SdkVersion = "v0.11.0";
```

Comment:
```suggestion
    public static OESdkVersion = "v0.11.0";
```

Reply:
This variable has been removed according to Brian's comment.